src/snowflake/snowpark/modin/plugin/_internal/join_utils.py

```diff
@@ -103,12 +103,55 @@ class JoinOrAlignInternalFrameResult(NamedTuple):
     result_column_mapper: JoinOrAlignResultColumnMapper
 
 
+def assert_snowpark_pandas_types_match(
+    left: InternalFrame,
+    right: InternalFrame,
+    left_identifiers: list[str],
+    right_identifiers: list[str],
+) -> None:
+    """
```

Comment:
i assume that is a direct code movement without change, is that right?

Reply:
Yes no code change beyond changing the variable names (using `left/right_identifiers` instead of `left/right_on`.